client/widgetdecorations.h

```diff
@@ -74,16 +74,38 @@ class fcwidget : public QFrame {
 class resizable_widget : public fcwidget {
   Q_OBJECT
 
+  static constexpr int event_width = 25;
+
 signals:
   void resized(QRect rect);
 
+public:
+  enum resizable_type {
+    top,
+    topLeft,
+    topRight,
+    bottom,
+    bottomLeft,
+    bottomRight,
+    left,
+    right
+  };
```

Comment:
Would this benefit from a `QFlags` declaration?

Reply:
> Would this benefit from a `QFlags` declaration?

Um, why?
What advantages does this give?

Reply:
You don't need to invent operations with the `flags` array and you can just OR enumerators like in C (but still in a type-safe manner). See https://doc.qt.io/qt-5/qflags.html#details

Reply:
> You don't need to invent operations with the `flags` array and you can just OR enumerators like in C (but still in a type-safe manner). See https://doc.qt.io/qt-5/qflags.html#details

done